src/gc/env/gcenv.base.h

```diff
@@ -220,34 +220,40 @@ typedef DWORD (WINAPI *PTHREAD_START_ROUTINE)(void* lpThreadParameter);
 
 #ifdef _MSC_VER
 #pragma intrinsic(_BitScanForward)
-#if WIN64
+#if _WIN64
  #pragma intrinsic(_BitScanForward64)
 #endif
 #endif // _MSC_VER
 
 // Cross-platform wrapper for the _BitScanForward compiler intrinsic.
+// A value is unconditionally stored through the `bitIndex` argument,
+// but callers should only rely on it when the function returns TRUE;
+// otherwise, the stored value is undefined and varies by implementation
+// and hardware platform.
 inline uint8_t BitScanForward(uint32_t *bitIndex, uint32_t mask)
 {
 #ifdef _MSC_VER
     return _BitScanForward((unsigned long*)bitIndex, mask);
 #else // _MSC_VER
-    unsigned char ret = FALSE;
     int iIndex = __builtin_ffsl(mask);
```

Comment:
Why `__builtin_ffsl` rather than `__builtin_ffs`?

The former works on `long` (which will be 64-bit on x64), while the latter works on `int`.

Reply:
https://godbolt.org/z/kaT5ZV

Reply:
Good point. I'll fix it.